cmd/cli/task_config/ask_plan.go

```diff
@@ -0,0 +1,38 @@
+package task_config
+
+import (
+	"errors"
+
+	"github.com/jinzhu/configor"
+	"github.com/sonm-io/core/proto"
+)
+
+const (
+	minRamSize    = 4 * 1024 * 1024
+	minCPUPercent = 1
+)
+
+func validate(ask *sonm.AskPlan) error {
```

Comment:
Make as method. Also use `Validate() error` signature for auto-validation when sending to/from gRPC.